modules/ui/sections/background_list.js

```diff
@@ -189,6 +192,36 @@ export function uiSectionBackgroundList(context) {
             .classed('imagery-show-icons', showIcons)
             .html(function(d) { return d.label(); });
 
+        enter
+            .append('button')
+            .attr('class', 'stamp background-favorite-button')
+            .classed('active', d => categoryMapping(d).id === 'favs')
+            .on('click', (_, d) => {
+                var group = categoryMapping(d);
+                if (group.id === 'favs') {
+                    delete _favouriteLayerIdMappings[d.id];
+                } else {
+                    _favouriteLayerIdMappings[d.id] = categoryGroups.favs;
+                }
+                prefs('background-favorites', JSON.stringify(Object.keys(_favouriteLayerIdMappings)));
+                _sources = updateSources();
+                reRender();
+                d3_selectAll('.layer-background-list li').filter(item => item === d)
+                    .transition()
+                    .duration(300)
+                    .ease(d3_easeCubicInOut)
+                    .style('background-color', '#ccc')
+                    .transition()
+                    .duration(300)
+                    .ease(d3_easeCubicInOut)
+                    .style('background-color', null);
+            })
+            .call(svgIcon('#maki-heart-11'))
```

Comment:
Now a layer could potentially be visually distinguished by activeness, category, favoriteness, geographic scope, quality, and (via the icon) provider. That’s a lot of facets to display, potentially making the UI look a bit busy. (But I did argue that mappers need to balance a lot of considerations when selecting imagery…) Maybe the “best” flag could be treated as a “favorite by default” flag instead, to streamline one of the facets.